package.json

```diff
@@ -60,11 +61,12 @@
     "build:types:lib": "vue-tsc -p tsconfig.types.json && tsc-alias -p tsconfig.types.json",
     "build:types:resolvers": "tsc -p tsconfig.resolvers.json --declaration --emitDeclarationOnly --outDir dist/types",
     "build:types:rules": "tsc -p tsconfig.rules.json --declaration --emitDeclarationOnly --outDir dist/types",
-    "build:types": "yarn build:types:lib && yarn build:types:resolvers && yarn build:types:rules",
+    "build:types": "yarn build:types:lib && yarn build:types:resolvers && yarn build:types:rules && yarn build:types:router",
     "build:theme": "zx scripts/build-theme.mjs && mkdir -p dist/assets/fonts && cp src/assets/fonts/* dist/assets/fonts && mkdir -p dist/styles && cp -r src/styles/* dist/styles",
     "build:icons": "zx scripts/build-icons.mjs",
     "build:resolvers": "vite build --config vite.config.resolvers.ts",
     "build:rules": "vite build --config vite.config.rules.ts",
+    "build:types:router": "vue-tsc -p tsconfig.router-types.json && tsc-alias -p tsconfig.router-types.json",
```

Comment:
in the file, we grouped the commands (the types ones in the same place) maybe it would be cleaner to organize this one like that